plugin/pkg/admission/limitranger/admission.go

```diff
@@ -467,19 +459,23 @@ func (d *DefaultLimitRangerActions) SupportsAttributes(a admission.Attributes) b
 }
 
 // SupportsLimit always returns true.
-func (d *DefaultLimitRangerActions) SupportsLimit(limitRange *api.LimitRange) bool {
+func (d *DefaultLimitRangerActions) SupportsLimit(limitRange *corev1.LimitRange) bool {
 	return true
 }
 
 // PersistentVolumeClaimValidateLimitFunc enforces storage limits for PVCs.
 // Users request storage via pvc.Spec.Resources.Requests.  Min/Max is enforced by an admin with LimitRange.
 // Claims will not be modified with default values because storage is a required part of pvc.Spec.
 // All storage enforced values *only* apply to pvc.Spec.Resources.Requests.
-func PersistentVolumeClaimValidateLimitFunc(limitRange *api.LimitRange, pvc *api.PersistentVolumeClaim) error {
+func PersistentVolumeClaimValidateLimitFunc(limitRange *corev1.LimitRange, pvc *api.PersistentVolumeClaim) error {
 	var errs []error
 	for i := range limitRange.Spec.Limits {
-		limit := limitRange.Spec.Limits[i]
-		limitType := limit.Type
+		coreLimit := api.LimitRangeItem{}
+		if err := apiscorev1.Convert_v1_LimitRangeItem_To_core_LimitRangeItem(&limitRange.Spec.Limits[i], &coreLimit, nil); err != nil {
+			return err
+		}
+		limit := coreLimit
+		limitType := coreLimit.Type
 		if limitType == api.LimitTypePersistentVolumeClaim {
 			for k, v := range limit.Min {
```

Comment:
`k` in this case could be cast to a string and then cast to the internal, right?  How about updating the minConstraint method to take it as a string